src/OrderFormAcceptanceTests.Actions/Collections/PageActionCollection.cs

```diff
@@ -7,5 +7,6 @@ public sealed class PageActionCollection
 		public Authentication Authentication { get; set; }
 		public Homepage Homepage { get; set; }
 		public OrderForm OrderForm { get; set; }
+		public Dashboard Dashboard { get; set; }
```

Comment:
Which dashboard is this?

Reply:
![image](https://user-images.githubusercontent.com/55191681/82301711-8998ae80-99b0-11ea-9754-4d67b9f4e0fc.png)
This one, /order/organisation

Reply:
Can this be reflected in the class name as there are likely to be sub-dashboards in the order form itself